src/payment/strategies/googlepay/googlepay-adyenv2-initializer.spec.ts

```diff
@@ -0,0 +1,38 @@
+import GooglePayAdyenV2Initializer from './googlepay-adyenv2-initializer';
+import { getAdyenV2PaymentDataMock, getAdyenV2PaymentDataRequest, getAdyenV2PaymentMethodMock, getAdyenV2TokenizedPayload, getCheckoutMock } from './googlepay.mock';
+
+describe('GooglePayAdyenV2Initializer', () => {
+    let googlePayInitializer: GooglePayAdyenV2Initializer;
+
+    beforeEach(() => {
+        googlePayInitializer = new GooglePayAdyenV2Initializer();
+    });
+
+    it('creates an instance of GooglePayAdyenV2Initializer', () => {
+        expect(googlePayInitializer).toBeInstanceOf(GooglePayAdyenV2Initializer);
+    });
+
+    describe('#initialize', () => {
+        it('initializes the google pay configuration for adyenv2', async () => {
+            const initialize = await googlePayInitializer.initialize(
+                getCheckoutMock(),
+                getAdyenV2PaymentMethodMock(),
+                false
+            );
+
+            expect(initialize).toEqual(getAdyenV2PaymentDataRequest());
+        });
+    });
+
+    describe('#teardown', () => {
+        it('teardown the initializer', () => expect(googlePayInitializer.teardown()).resolves.toBeUndefined());
```

Comment:
`it returns a promise that always resolves?`

also, even though the test itself is one line, might be better to keep the title and the test in different lines.